core/state/trie_prefetcher.go

```diff
@@ -336,11 +336,7 @@ func (sf *subfetcher) loop() {
 					if _, ok := sf.seen[string(task)]; ok {
 						sf.dups++
 					} else {
-						if len(task) == len(common.Address{}) {
-							sf.trie.TryGetAccount(task)
-						} else {
-							sf.trie.TryGet(task)
-						}
+						sf.trie.TryGet(task)
```

Comment:
Wait, how can this change be correct?

Reply:
It's only for prefetching. 

`TryGetAccount` will load the node and decode it as an account
`TryGet` will only load the node

Actually they are equivalent.